test-review/apps/session/src/test/java/com/redhat/training/conference/session/SessionResourceTest.java

```diff
@@ -16,6 +17,7 @@
 @QuarkusTest
 public class SessionResourceTest {
 
+    @RestClient
```

Comment:
Without the annotation the tests are not going to pass because junit is not able to find the associated bean.